### PR TITLE
fix null check

### PR DESCRIPTION
**Pull request by aghassemi**
Fixes https://github.com/ampproject/amphtml/issues/15684

### CODE REVIEW

File: src/service/standard-actions-impl.js [hunk 1]
```diff
@@ -113,7 +113,7 @@ export class StandardActions {
       case 'navigateTo':
         // Some components have additional constraints on allowing navigation.
         let permission = Promise.resolve();
-        if (startsWith(node.tagName, 'AMP-')) {
+        if (node.tagName && startsWith(node.tagName, 'AMP-')) {
```
Comment (aghassemi) [inline src/service/standard-actions-impl.js]: @choumx Another thing I noticed is that `node` here (target of invocation) is always "document" when an action is a global action called via `AMP.` and per https://github.com/ampproject/amphtml/blob/master/src/service/action-impl.js#L478 that seems to be the desired behaviour. So I am wondering if this line is even necessary. ?

Comment (choumx) [reply src/service/standard-actions-impl.js]: It's should be checking `invocation.caller.tagName`. #15693 fixes it.

**Comment by choumx** [comment]
Thanks for jumping on this bug!

**Comment by aghassemi** [comment]
Closing in favour of the correct fix here: #15693

**Pull request closed without merging**